### PR TITLE
Add release notes for v0.8.2 and bump versions

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: website/blog/2020-03-19-cobalt.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.1 - Cobalt
```
Comment (gabro) [inline website/blog/2020-03-19-cobalt.md]: 0.8.2

Comment (tgodzik) [reply website/blog/2020-03-19-cobalt.md]: Fixed, no one saw that :smile:

File: website/blog/2020-03-19-cobalt.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.2 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are happy to announce the release of Metals v0.8.2, which main purpose is
+adding support for new Scala version 2.12.11. Additionally, we included a couple
```
Comment (ckipp01) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
adding support for new the Scala version, 2.12.11. Additionally, we included a couple
```

File: website/blog/2020-03-19-cobalt.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
```
Comment (ckipp01) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
```

File: website/blog/2020-03-19-cobalt.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+## Miscalenous improvements
+
+- handle fatal exceptions when evaluating worksheets
+- fix issue with false positive warning about Bloop version change
+- fix rename for hierarchy with generics
+- ignore return type when renaming override methods and fields
+- fix issue when docstrings where not update after changes in the workspace
+- it's now possible to navigate to Scala sources even outside a build target or
+  in a worksheet
+
```
Comment (ckipp01) [inline website/blog/2020-03-19-cobalt.md]: - fix issue with string alignment when multiple multiline strings were present in a file

File: website/blog/2020-03-19-cobalt.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
+give Metals a try!
+
+- support for Scala 2.12.11
+- recent minor improvements
+
+## Miscalenous improvements
```
Comment (olafurpg) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
## Miscellaneous improvements
```

File: website/blog/2020-03-19-cobalt.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+- handle fatal exceptions when evaluating worksheets
```
Comment (olafurpg) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
- fix bug where worksheets got stuck evaluating forever
```

File: website/blog/2020-03-19-cobalt.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+- fix issue with false positive warning about Bloop version change
```
Comment (olafurpg) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
- fix issue where Metals would incorrectly prompt about a Bloop version change
```

File: website/blog/2020-03-19-cobalt.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+- fix rename for hierarchy with generics
```
Comment (olafurpg) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
- fix a bug where rename symbol produced invalid code for class hierarchies using generics
```

File: website/blog/2020-03-19-cobalt.md [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+- ignore return type when renaming override methods and fields
```
Comment (olafurpg) [inline website/blog/2020-03-19-cobalt.md]: In what way could the user have observed this issue?

File: website/blog/2020-03-19-cobalt.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+- fix issue when docstrings where not update after changes in the workspace
```
Comment (olafurpg) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
- fix bug where docstrings for workspace sources returned stale documentation
```

File: website/blog/2020-03-19-cobalt.md [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+- it's now possible to navigate to Scala sources even outside a build target or
```
Comment (olafurpg) [inline website/blog/2020-03-19-cobalt.md]:
```suggestion
- goto definition now works for standalone source files, even if the build has not been imported
```

File: website/blog/2020-03-19-cobalt.md (hunk 5 above)
Comment (tgodzik) [reply website/blog/2020-03-19-cobalt.md]: Haha,  I shouldn't use that word, I always have problems with it :smile:

File: website/blog/2020-03-19-cobalt.md (hunk 9 above)
Comment (tgodzik) [reply website/blog/2020-03-19-cobalt.md]: In a situation like:
```
trait A
trait B extends A

trait C{
  def method : A
}

trait D extends C{
  override def method : B
}

```

**Comment by tgodzik** [comment]
> A few suggestions on how to formulate the changes in terms of user-facing behavior. Maybe we should start a new convention to do the same for PR titles

I just copied the PR titles, which might not have been the best idea. For sure it would be easier to do the release with more sensible user facing titles. We could tag every user facing PR and make sure it is properly named?

File: website/blog/2020-03-19-cobalt.md (hunk 11 above)
Comment (tgodzik) [reply website/blog/2020-03-19-cobalt.md]: Thanks, fixed all of the titles!

**Pull request merged**